### DISCUSSION TITLE
Add the ability to change the view between card and list views

### DISCUSSION THREAD

**Discussion by DatuStram** [gotson/komga#1573] [Ideas]
### Describe your suggested feature

It would be great if there was the possibility to change the view between the current "card" view and a "list" view, displaying smaller cover thumbnails alongside some metadata such as writer, artists, release date, the book's summary,...

Ideally this would be available in series view, collection view and reading list view.

### Other details

Maybe you could take some inspiration from the way it's done in [Kavita](https://www.kavitareader.com/)?
I really **DO** prefer the way Komga does things in general, but I sorely miss this feature 😅

![kavita_covers@0 75x](https://user-images.githubusercontent.com/24694690/221333633-a9f1f249-0f90-4c87-ab36-be842d9c5310.png)
![kavita_list@0 75x](https://user-images.githubusercontent.com/24694690/221333636-378c618e-4ca1-4d40-b6a9-bc46d64ae288.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
I was checking on this in Kavita and couldn't find the option in their demo server, does it still exist ?

**Reply by gotson** [reply]
Kavita removed this a while back, but Komelia has a similar view:
![komelia list view](https://github.com/user-attachments/assets/27666c70-6f6f-41d8-a452-5103b2647772)

**Comment by sam5558** [comment]
Thank you for the great work @gotson .

Having the choice to change the display layout is really a must-have.
Either on the library books or within a book, this should be available.

**Comment by gotson** [comment]
Would need to be enabled for readlist content to enable easier ordering: https://github.com/gotson/komga/discussions/2290

**Reply by FlyveHest** [reply]
Personally I've only really missed it for ordering, the tiled look is fine for browsing I think.

But having a toggle on a given view for tiled / list view would be fantastic.
